### PR TITLE
Improve permissions aa1 to implement 1

### PR DESCRIPTION
**Pull request by Aichnerc**
**Checklist:**

 

-   [x] Have you added an explanation of what your changes do and why you'd like them to be included?
-   [ ] Have you updated or added documentation for the change?
-   [x] Have you tested your changes with successful results?

**Type of Changes:**

 

-   [ ] Bug fix (non-breaking change which fixes an issue)
-   [x] New feature (non-breaking change which adds functionality)
-   [ ] Documentation (non-breaking change which adds documentation)
-   [ ] Breaking change (fix or feature that would cause existing functionality to change)

**What is the current behavior? (link to any open issues here)**

- Currently it is not possible to edit groups, create groups.
- Currently it is not possible to set permissions for groups.

**What is the new behavior (if this is a feature change)?**

- It is now to edit and create groups.
- Permissions have been introduced and can be set when editing or creating a group.

**Known issues**:

- When creating a group, permissions that have not been checked are not being initialized. This leads to the permission type not being set as state and therefore not having a `false` value. This should not be a breaking, but will be fixed on a later date by @Aichnerc

### CODE REVIEW

**Review by schettn — APPROVED** [review]
lgtm

File: src/components/organisms/sections/Permissions/index.jsx [hunk 1]
```diff
@@ -167,13 +185,56 @@ class Permissions extends React.Component {
...
+      : [];
+
+    // Remove type to change from permission types
+    let otherTypes = currentPermission[0]
+      ? currentPermission[0].types.filter(
+          (item) => this.unifyString(item.name) !== this.unifyString(type.name)
+        )
+      : [];
+
+    // Readd updated item to change
```
Comment (kleberbaum) [inline src/components/organisms/sections/Permissions/index.jsx]:
```suggestion
    // Read updated item to change
```

**Review by kleberbaum — CHANGES_REQUESTED** [review]
Should be ok to merge after a few changes.

**Review by kleberbaum — APPROVED** [review]
Should be ok to merge.

**Pull request merged**